presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -286,6 +292,9 @@ private void printQueryInfo(QueryResults results)
             verify(terminalWidth >= 75); // otherwise handled above
             int progressWidth = (min(terminalWidth, 100) - 75) + 17; // progress bar is 17-42 characters wide
 
+            previousStmtStats = previousStmtStats == null ? new PreviousStageStatus(start, 0, 0) : previousStmtStats;
```

Comment:
Usually we do not use abbreviation or '? + :' combinations.
Let's spell out Stmt -> Statement.
just inline 'new PreviousStageStatus(start, 0, 0)' to its declaration; not necessary to check null